VideoLocker/src/main/java/org/edx/mobile/player/VideoListFragment.java

```diff
@@ -441,12 +440,8 @@ public void onSelectItem() {
             videoListView.setOnItemClickListener(adapter);
 
             setActivityTitle(enrollment.getCourse().getName());
-            try{
-                environment.getSegment().screenViewsTracking("My Videos - All Videos - "
-                        + enrollment.getCourse().getName());
-            }catch(Exception e){
-                logger.error(e);
-            }
+            environment.getSegment().trackScreenView("My Videos - All Videos - "
+                    + enrollment.getCourse().getName());
```

Comment:
Unfortunately, there's no list, they should really be broken out into an enum or something at the least. And that is indeed a terrible name, but it wasn't a task right now to go through and fix up all the names, just the ones on the assessments path.